tests/spread/plugins/scons/snaps/scons-hello/snap/snapcraft.yaml

```diff
@@ -1,22 +1,22 @@
 name: scons-hello
-version: 0.1
+version: '1.0'
 summary: test the scons plugin
 description: |
   This is a basic scons snap. It just prints a hello world.
   If you want to add other functionalities to this snap, please don't.
   Make a new one.
-icon: icon.png
-confinement: strict
 
-build-packages: [gcc, libc6-dev]
+grade: devel
+confinement: strict
 
 apps:
   scons-hello:
-    command: opt/bin/main
+    command: hello
 
 parts:
   scons-project:
-    scons-options:
-      - --prefix=/opt
     plugin: scons
     source: .
+    scons-options:
+      - --prefix=/
+    build-packages: [gcc, libc6-dev]
```

Comment:
is libc6-dev really needed explicitly?

Reply:
I didn't check, honestly, I just assumed so since this is only a slight modification of an existing test.